myokit/_model_api.py

```diff
@@ -4357,10 +4359,11 @@ def lhs(self):
         """
         return self._lhs
 
-    def promote(self, state_value=0):
+    def promote(self, state_value=0, check_const=True):
         """
         Turns this variable into a state variable with a current state value
-        given by ``state_value``.
+        given by ``state_value``. state_value could be a value, an Expression,
+        or a proto expression
```

Comment:
Oh no! You are the first person ever to discover the dirty secret that is proto expressions 🙈 

Maybe omit them here for clarity? A user using Myokit in a non-hacky way shouldn't have to know about them, I think.

Reply:
done